src_cam/physpkg.F90

```diff
@@ -2915,6 +3057,14 @@ subroutine tphysbc (ztodt, state,  &
       prec_str = 0._r8
       snow_str = 0._r8
 
+      if (compute_enthalpy_flux) then
+         ! In first time-step tphysac variables need to be zero'd out
+         if (compute_enthalpy_flux) then
```

Comment:
Something seems wrong here since the inner `if` statement is the same as the outer one. The comment mentions "first time-step" but the logic does not reflect that.